README.md

```diff
@@ -84,6 +94,20 @@ Run `yarn test`. To start a test watcher, run `yarn run test:watch`.
 
 Run `yarn run` to see other available commands.
 
+## Environment variables
+
+- **NODE_ENV**: Used to set the environment. Set it to 'production' to enable optimizations. Defaults to 'development'.
+- **LUCIFY_ENV**: Overrides NODEENV if set. Meant to be used for automatic deployments.
```

Comment:
Left th underscore out because it broke VSCode's Markdown parsing, but I guess I could add it